### PR TITLE
Added Movement Speed Multiplier Effects

### PR DESCRIPTION
**Pull request by OnlyRealNubs**
These consist of:
```
10x Movement Speed
5x Movement Speed
0.5x Movement Speed
```

### CODE REVIEW

File: ChaosMod/Effects/db/Player/PlayerMovementSpeed.cpp [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+REGISTER_EFFECT(nullptr, nullptr, OnTickx10, EffectInfo
+	{
+		.Name = "10x Movement Speed",
+		.Id = "player_movementx10",
+		.IsTimed = true,
+		.IsShortDuration = true,
+		.EffectCategory = EEffectCategory::Movement
+	}
+);
+
```
Comment (Last0xygen) [inline ChaosMod/Effects/db/Player/PlayerMovementSpeed.cpp:36]: missing clang on 😏

**Review by Last0xygen — CHANGES_REQUESTED** [review]

**Comment by pongo1231** [comment]
They're missing from the config app

**Comment by OnlyRealNubs** [comment]
Fixed

**Comment by pongo1231** [comment]
5x Movement Speed and 10x Movement Speed are marked as short timed in the mod but not in the config app.

I'd probably just make both of them normal timed in the mod.

**Comment by OnlyRealNubs** [comment]
Ahh. I read only 0.5 as short.  Fixed now.

**Pull request merged**